pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[tool.pdm]
+version = { source = "file", path = "src/nuitka_tasks/version.py" }
+allow_prereleases = true
+
+[project]
+name = "nuitka-tasks"
+dynamic = ["version"]
+authors = [
+    {name = "xinetzone", email = "xinzone@outlook.com"},
+]
+dependencies = [
+  "importlib-metadata>=4.12",
+  "unidecode",
+  "lxml",
+  "requests",
+  "rstfmt==0.0.11",
+  "setuptools",
+  "wheel",
+]
+requires-python = ">=3.10"
+readme = "README.rst"
+license = {file = "LICENSE"}
+
+[build-system]
+requires = ["pdm-pep517"]
+build-backend = "pdm.pep517.api"
+
+[project.urls]
+Home = "http://nuitka.net/"
+
+# [project.entry-points]
+# "sphinx.html_themes" = {tvm_book = "tvm_book"}
+
+[project.optional-dependencies]
+doc = [
+  "myst-nb",
+  # "xyzstyle",
+  "matplotlib",
+  "sphinx-copybutton",
+  "sphinxcontrib-bibtex",
+  "sphinx-thebe",
+  "sphinx-design",
+  "sphinx-automodapi",
+  "sphinx-sitemap",
+  "sphinx_plotly_directive",
+  "sphinx-comments",
+  "sphinxcontrib-mermaid",
+  "sphinxcontrib-youtube",
+  "sphinx-favicon",
+  "sphinx_rtd_theme",
+  "ablog",
+  # "plotly"
+]
+
+[tool.pdm.build]
+package-dir = "src"
+# includes = []
+# excludes = []
+# source-includes = ["tests/"]
+
+[pypi.extra]
+url = "https://pypi.tuna.tsinghua.edu.cn/simple "
```

Comment:
What is this one for?

Reply:
Provide additional PyPI index library, for China region, can speed up `pip install`.